metron-analytics/metron-profiler/src/main/java/org/apache/metron/profiler/bolt/ProfileBuilderBolt.java

```diff
@@ -194,8 +194,8 @@ private void init(Tuple input) {
       expressions.forEach((var, expr) -> executor.assign(var, expr, message));
 
     } catch(ParseException e) {
-      String msg = format("Bad 'init' expression: %s, profile=%s, entity=%s, start=%d",
-              e.getMessage(), measurement.getProfileName(), measurement.getEntity(), measurement.getStart());
+      String msg = format("Bad 'init' expression: %s, profile=%s, entity=%s",
+              e.getMessage(), measurement.getProfileName(), measurement.getEntity());
```

Comment:
Yes, they already are LOG.error'd inside `execute`.  

The point of this catch is to add the extra note that the failure came from the 'init' expression, rather than one of the other expressions involved in creating a profile.  I wanted to make it blatantly obvious to the user which expression failed.